clsCommon.pri

```diff
@@ -14,7 +14,6 @@ HEADERS *=\
 	source/beamline/CLS/CLSValves.h \
 	source/beamline/CLS/CLSShutters.h \
 	source/beamline/CLS/CLSMirrorMaskState.h \
-	source/beamline/CLS/CLSSSRLMonochromatorMaskState.h \
```

Comment:
Is this something which got moved up to a CLS class in the recent refactor, but didn't really make sense as a CLS class?